server/src/test/java/org/apache/druid/client/cache/MemcachedCacheTest.java

```diff
@@ -224,6 +217,37 @@ public void emit(Event event)
     }
   }
 
+  @Test
+  public void testSslConnection()
+  {
+    final MemcachedCacheConfig config = new MemcachedCacheConfig()
+    {
+      @Override
+      public boolean enableTls()
+      {
+        return true;
+      }
+
+      @Override
+      public String getHosts()
+      {
+        return "localhost:9999";
+      }
+    };
+    // Static Mode
+    Assert.assertEquals(config.getClientMode(), "static");
```

Comment:
this test seems to ensure that the default client mode is static, which is fine, but it doesn't actually do what the test name suggests, which is to verify that ssl is configured. Can we move this to its own test to verify the default?